packages/components/package.json

```diff
@@ -65,6 +66,7 @@
 		"@wordpress/primitives": "^4.23.0",
 		"@wordpress/react-i18n": "^4.23.0",
 		"@wordpress/url": "^4.23.0",
+		"@wordpress/warning": "3.23.0",
```

Comment:
Kept `@wordpress/warning` for now, although we should understand the general level of coupling with WordPress going forward.